src/App.tsx

```diff
@@ -1,61 +1,41 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { NewTodo } from './components/NewTodo';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoUser } from './types/TodoUser';
+
+function assignData(users: User[], todos: Todo[]): TodoUser[] {
+  const newData: TodoUser[] = [];
+
+  users.forEach(user => {
+    todos.forEach(todo => {
+      if (user.id === todo.userId) {
+        newData.push(Object.assign({ ...todo }, { user: user }));
+      }
+    });
+  });
+
+  return newData;
+}
+
+const list = assignData(usersFromServer, todosFromServer);
```

Comment:
[CODE STYLE] - Don't repeat yourself. If you see that you copy/paste some block code, maybe, it's time to create a separate variable/function. You are using `assignData(usersFromServer, todosFromServer)` in two places, consider creating a separate variable for it.